apps/ledger-live-mobile/src/families/ethereum/EditTransaction/EditOperationCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { SideImageCard } from "@ledgerhq/native-ui";
+import { Operation } from "@ledgerhq/types-live";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import SectionContainer from "../../../screens/WalletCentricSections/SectionContainer";
+
+type EditOperationCardProps = {
+  oldestEditableOperation: Operation;
+  isOperationStuck: boolean;
+  onEditTransactionPress: (operation: Operation) => void;
+};
+
+export const EditOperationCard = ({
+  oldestEditableOperation,
+  isOperationStuck,
+  onEditTransactionPress,
+}: EditOperationCardProps) => {
+  const { t } = useTranslation();
+  const flag = useFeature("editEthTx");
+
+  return flag?.enabled ? (
+    <SectionContainer px={6}>
+      <SideImageCard
+        title={t(
+          isOperationStuck
```

Comment:
I don't know whether I am wrong. Is your EditOperationCard the warning message panel on top of the operation list? 
If so, we shouldn't display it when it is not a stuck operation. Right?

Reply:
from the design I have we show a message before it's considered stuck as well